app/settings.py

```diff
@@ -75,6 +75,7 @@ def get_env_or_fail(key):
 EQ_DEV_MODE = parse_mode(os.getenv('EQ_DEV_MODE', 'False'))
 EQ_ENABLE_CACHE = parse_mode(os.getenv('EQ_ENABLE_CACHE', 'True'))
 EQ_ENABLE_FLASK_DEBUG_TOOLBAR = parse_mode(os.getenv('EQ_ENABLE_FLASK_DEBUG_TOOLBAR', 'False'))
+EQ_SECURE_SESSION_COOKIE = parse_mode(os.getenv('EQ_SECURE_SESSION_COOKIE', 'False'))
```

Comment:
Just looking at this again we should probably follow the convention of having `ENABLE` in the name as its a boolean value.

Reply:
We should also use a production default and just override in Dev

Reply:
Ah, that was daft of me. Good catch!